tests/image/test_list.py

```diff
@@ -107,17 +110,17 @@ def test_run_cmd_in_docker_container(get_docker_client_mock):
         stream=False, timestamps=False)
 
 
+@pytest.mark.parametrize('is_binary_explanation', [True, False])
 @mock.patch('shub.image.list._get_project_settings', return_value={})
 @mock.patch('shub.image.utils.get_docker_client')
-def test_shub_image_info_fallback(get_docker_client_mock, _):
-    exception = docker.errors.APIError(
-        mock.Mock(),
-        mock.Mock(),
-        explanation=(
-            'Cannot start container xxx: [8] System error: exec: "shub-image-info": '
-            'executable file not found in $PATH'
-        )
-    )
+def test_shub_image_info_fallback(get_docker_client_mock, _,
+                                  is_binary_explanation):
+    error_msg = ('Cannot start container xxx: [8] System error: exec:'
+                 ' "shub-image-info": executable file not found in $PATH')
+    if is_binary_explanation:
+        error_msg = error_msg.encode('utf-8')
```

Comment:
Same as above, `error_msg` is already `str` (but we can fix it easily by adding a `u` prefix)